src/main.rs

```diff
@@ -42,6 +43,8 @@ fn show_version() {
     println!("{}", version_info);
 }
 
+include!("docs.rs");
```

Comment:
If possible I would prefer to use a module with `mod docs` as rust-analizer still is a bit iffy with files included differently AFAIK

Reply:
done.